### PR TITLE
fix: Removed embed meeting option from toolbar

### LINKED ISSUE #6487: Virtual Events: Hide "Embed" option on Jitsi Calls
The embed option in Jitsi is a feature that we do not really want here as it would allow others to embed a call elsewhere. Please try to hid the functionality. Can this be achieved even without having access to Jitsi itself, e.g. with CSS?

Screenshot from 2021-01-25 22-52-36

### PR DESCRIPTION
**Pull request by Creatoon**
Fixes #6487 

#### Short description of what this resolves:
This PR removes embed meeting option from toolbar from the jitsi meeting call.

we can see that option is removed now.
Screenshot from 2021-01-27 01-06-56

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Creatoon** [comment]
@iamareebjamal Sir i tried first removing it with css , as mentioned in the issue , but we can't do that as meeting is being rendered on an iframe, that's why lastly i came up with this solution, please review , Thanks.

**Comment by iamareebjamal** [comment]
Instead of listing the allowed buttons. Remove the unallowed ones by excluding from default list

**Comment by Creatoon** [comment]
> Instead of listing the allowed buttons. Remove the unallowed ones by excluding from default list

Sir , i reffered the docs and there is no way to exclude some fields from the default toolbar list, the only way given is to override the default one.

**Comment by iamareebjamal** [comment]
Get the default list and filter

**Comment by Creatoon** [comment]
@iamareebjamal sir I think,  this pr is completed Now,  you can review it,  thanks

File: app/components/public/stream/jitsi-stream.ts [hunk 1]
```diff
@@ -23,6 +23,14 @@ export default class PublicStreamJitsiStream extends Component<Args> {
     const stream = this.args.videoStream;
     const channel = await stream.videoChannel;
 
+    const defaultToolbarButtons = [
+      'microphone', 'camera', 'closedcaptions', 'desktop', 'embedmeeting', 'fullscreen',
+      'fodeviceselection', 'hangup', 'profile', 'chat', 'recording',
+      'livestreaming', 'etherpad', 'sharedvideo', 'settings', 'raisehand',
+      'videoquality', 'filmstrip', 'invite', 'feedback', 'stats', 'shortcuts',
+      'tileview', 'videobackgroundblur', 'download', 'help', 'mute-everyone', 'security'
+    ];
```
Comment (iamareebjamal) [inline app/components/public/stream/jitsi-stream.ts:32]: Get default list from jitsi

**Comment by Creatoon** [comment]
@iamareebjamal sir , i found that , toolbar buttons and every interface configuration is being sent with the html page itself, inside script tag, they are not getting updated from the api or jitsiMeetExternalApi, also i asked this question in the community of jitsi and they replied there is no soch option avilable this time.

Screenshot_2021-01-29 Is jitsi capable running call ON and ON - Developers Jitsi Meet API - Jitsi Community Forum - develop

File: app/components/public/stream/jitsi-stream.ts [hunk 2]
```diff
@@ -23,6 +23,14 @@ export default class PublicStreamJitsiStream extends Component<Args> {
     const stream = this.args.videoStream;
     const channel = await stream.videoChannel;
 
+    const defaultToolbarButtons = [
+      'microphone', 'camera', 'closedcaptions', 'desktop', 'embedmeeting', 'fullscreen',
```
Comment (iamareebjamal) [inline app/components/public/stream/jitsi-stream.ts:27]:
```suggestion
      'microphone', 'camera', 'closedcaptions', 'desktop', 'fullscreen',
```

File: app/components/public/stream/jitsi-stream.ts [hunk 3]
```diff
@@ -44,7 +52,8 @@ export default class PublicStreamJitsiStream extends Component<Args> {
         startWithVideoMuted : true
       },
       interfaceConfigOverwrite: {
-        HIDE_INVITE_MORE_HEADER: true
+        HIDE_INVITE_MORE_HEADER : true,
+        TOOLBAR_BUTTONS         : defaultToolbarButtons.filter(el => el !== 'embedmeeting')
```
Comment (iamareebjamal) [inline app/components/public/stream/jitsi-stream.ts:56]:
```suggestion
        TOOLBAR_BUTTONS         : defaultToolbarButtons
```

**Pull request merged**